mypy/build.py

```diff
@@ -1353,6 +1367,18 @@ def strongly_connected_components_path(vertices: Set[str],
                                        edges: Dict[str, List[str]]) -> Iterator[Set[str]]:
     """Compute Strongly Connected Components of a graph.
 
+    The graph is a DAG.
```

Comment:
You're reviewing an out-of-date version of this file. When updating the docstrings I realized that this is nonsense and corrected it.